src/sysupdate/sysupdate.c

```diff
@@ -694,6 +723,7 @@ static int context_show_version(Context *c, const char *version) {
                                           SD_JSON_BUILD_PAIR_BOOLEAN("installed", FLAGS_SET(us->flags, UPDATE_INSTALLED)),
                                           SD_JSON_BUILD_PAIR_BOOLEAN("obsolete", FLAGS_SET(us->flags, UPDATE_OBSOLETE)),
                                           SD_JSON_BUILD_PAIR_BOOLEAN("protected", FLAGS_SET(us->flags, UPDATE_PROTECTED)),
+                                          SD_JSON_BUILD_PAIR_STRV("changelog-url", changelog_urls),
```

Comment:
perhaps should be renamed `changelog-urls` instead of `changelog-url`? Ditto w/ appstream. Then make corresponding changes in sysupdated/gnome-software plugin/etc